pkg/armhelpers/interfaces.go

```diff
@@ -74,10 +92,10 @@ type AKSEngineClient interface {
 	DeleteVirtualMachine(ctx context.Context, resourceGroup, name string) error
 
 	// ListVirtualMachineScaleSets lists the vmss resources in the resource group
```

Comment:
let's standardize all the references to vmss/VMSS/Vmss to `VMSS` in the comments/code/tests

Reply:
There are a lot of places across the code-base using different casing, so let's do this in a different PR

Reply:
I don't see why we wouldn't do that in this PR.

Reply:
to clarify, I meant the references added by this PR, not all the references in the codebase :)

Reply:
I meant I will send a separate PR that fixes all of them across the code base.
but if you insist on doing this one here, it's coming

Reply:
fixing them across the codebase would be 💯if you want to do that but not adding new inconsistencies in this PR is the low hanging fruit and was the aim of my comment.

Reply:
done